src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Square extends AbstractFigure {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
```

Comment:
The `getArea` method should not be in the same interface as `draw`, according to the single responsibility principle. Consider separating these into different interfaces.